docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,251 @@
+
+## Combining spansets
+
+Spanset operators let you combine two sets of spans using and (`&&`) as well as union (`||`).
+
+- `{condA} && {condB}`
+- `{condA} || {condB}`
+
+
+For example, if you want to find a trace that went through two specific regions:
+
+```
+{ .region = "eu-west-0" } && { .region = "eu-west-1" }
+```
+
+Note the difference between the above and the following:
+
+```
+{ .region = "eu-west-0" && .region = "eu-west-1" }
+```
+
+The second expression returns no traces because it's impossible for a single span to have a `.region` attribute that is set to both region values at the same time.
+
+## Aggregators
+
+So far, all of the example queries expressions have been about individual spans. You can use aggregate functions to ask questions about a set of spans. These currently consist of:
+
+- `count` - The total count across a given intrinsic or items.
+- `avg` - The average across a given intrinsic or items.
+
+Aggregate functions allow you to carry out operations on matching results to further refine the traces returned. For more information on planned future work, refer to [How TraceQL works]({{< relref "architecture" >}}).
+
+For example, to find traces where the total number of spans is greater than `10`:
+
+```
+count() > 10
+```
+
+Find traces where the average duration of the spans in a trace is greater than `20ms`:
+
+```
+avg(duration) > 20ms
+```
+
+For example, find traces that have more than 3 spans with an attribute `http.status` with a value of `200`:
+
+```
+{ .http.status = 200 } | count() > 3
+```
+
+## Examples
+
+Find any trace with a span attribute or resource attribute `namespace` set to `prod`:
+
+```
+{ .namespace = "prod" }
+```
+
+Find any trace with a span attribute `namespace` set to `prod`:
+
+```
+{ span.namespace = "prod" }
+```
+
+Find any trace with a `namespace` attribute set to `prod` and `http.status` attribute set to `200`:
+
+```
+{ .namespace = "prod" && .http.status = 200 }
+```
+
+Find any trace where spans within it have a `namespace` attribute set to `prod` and an `http.status` attribute set to `200`. In previous examples, all conditions had to be true on one span. These conditions can be true on either different spans or the same spans.
+
+```
+{ .namespace = "prod" } && { .http.status = 200 }
+```
+
+Find any trace where any span has an `http.method` attribute set to `GET` as well as a `status` attribute set to `ok`, where any other span also exists that has an `http.method` attribute set to `DELETE`, but does not have a `status` attribute set to `ok`:
+
+```
+{ .http.method = "GET" && status = ok } && { .http.method = "DELETE" && status != ok }
+```
+
+Find any trace where the average client-span duration in a trace exceeds a threshold of one second:
+
+```
+{ span.kind = "client" } | avg(duration) > 1s
```

Comment:
this may be a miss but i don't think traceql can currently query the span kind intrinsic. let's remove this example.

Reply:
Done.